Makefile

```diff
@@ -55,16 +66,8 @@ package-%:
 
 	@mv -v ../delphix-platform-$*_*_amd64.deb artifacts
 
-SHELL_SCRIPTS := \
-	debian/postinst \
-	debian/prerm \
-	usr/bin/download-latest-image \
-	usr/bin/unpack-image \
-	var/lib/delphix-platform/ansible/apply \
-	var/lib/delphix-platform/os-migration
-
 shellcheck:
-	shellcheck $(SHELL_SCRIPTS)
+	shellcheck $$(shfmt -f .)
```

Comment:
After a build, will this still work? It won't traverse down into the "stage" directory?

Reply:
I've tested it, and yeah, it will traverse into the staging directories. Is this an issue?